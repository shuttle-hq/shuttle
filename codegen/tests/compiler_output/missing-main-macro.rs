#[shuttle_infra()]
fn main() {}
