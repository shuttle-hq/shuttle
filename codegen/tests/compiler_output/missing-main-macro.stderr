error: cannot find attribute `shuttle_infra` in this scope
 --> tests/compiler_output/missing-main-macro.rs:1:3
  |
1 | #[shuttle_infra()]
  |   ^^^^^^^^^^^^^
